builtin/logical/database/rotation_test.go

```diff
@@ -998,6 +999,34 @@ func TestBackend_StaticRole_Rotation_MongoDBAtlas(t *testing.T) {
 	})
 }
 
+// TestQueueTickIntervalKeyConfig tests the configuration of queueTickIntervalKey
+// Previous to a fix, this test panicked.
+func TestQueueTickIntervalKeyConfig(t *testing.T) {
```

Comment:
I'm not sure I completely understand this, so apologies if this is a foolish one.

I was expecting to see some output in the logs when I ran this test locally to say it was using the default value for `0` and `-1` cases,

e.g. `bad value for...` from: https://github.com/hashicorp/vault/blob/05b37831c4e470ef9fa18f8444d97491204648fb/builtin/logical/database/rotation.go#L592

but it just says the context is cancelled.

![image](https://github.com/hashicorp/vault/assets/487783/9266bacf-a315-4df5-ad14-2881b1215ad4)

Is that what's expected?

Reply:
I wrote this test before my fix, and what happened for me is the test failed with a panic